test/s3-benchmarks/src/main/java/software/amazon/awssdk/s3benchmarks/TransferManagerDownloadBenchmark.java

```diff
@@ -15,70 +15,79 @@
 
 package software.amazon.awssdk.s3benchmarks;
 
+import static software.amazon.awssdk.s3benchmarks.BenchmarkUtils.BENCHMARK_ITERATIONS;
+import static software.amazon.awssdk.s3benchmarks.BenchmarkUtils.printOutResult;
 import static software.amazon.awssdk.utils.FunctionalUtils.runAndLogError;
 
 import java.io.File;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import software.amazon.awssdk.core.async.AsyncResponseTransformer;
 import software.amazon.awssdk.services.s3.model.GetObjectRequest;
-import software.amazon.awssdk.transfer.s3.Download;
 import software.amazon.awssdk.utils.Logger;
 
 public class TransferManagerDownloadBenchmark extends BaseTransferManagerBenchmark {
     private static final Logger logger = Logger.loggerFor("TransferManagerDownloadBenchmark");
+    private final long contentLength;
 
     public TransferManagerDownloadBenchmark(TransferManagerBenchmarkConfig config) {
         super(config);
+        this.contentLength = s3Sync.headObject(b -> b.bucket(bucket).key(key)).contentLength();
```

Comment:
I think we should parameterize (i.e., `@Param`) the file/object size. WDYT?

Reply:
I think different object sizes may have very different performance profiles, and it would be helpful to be able to use this benchmark to determine at what thresholds the CRT/v1 behave differently, or when TransferManager becomes faster than a standard getObject request (TransferManager may be slower for very small objects, for example). The CRT input params should not be a constraint here. We can do something like this:

```
//       1KB       1MB        10MB
@Param({"1024", "1048576", "10485760"})
public long objectSize;

@Setup(Level.Invocation)
public void setUp() {
    // put object with objectSize
}

@Benchmark
public void benchmark() {
  // get object
}
```